### DISCUSSION TITLE
ESLint configuration

### DISCUSSION THREAD

**Discussion by MichalOsadowski** [zed-industries/zed#6547] [Feature Requests]
Hi there!
 
Found this editor tomorrow and it looks very promising. 

One thing i'm really missing from webstorm is eslint and prettier autofix on save and highlightning errors/warnings in code. 

Is it possibel to configure Zed to work in similar way?

**Comment by lunelson** [comment]
This is one of my top questions too. Zed is very compelling, I love the speed and the tree-sitter based system rather than AST; but I work in a team which needs to have quite comprehensive linting checks, and although I could obviously run them separately, I've grown pretty dependent on having them inline with VSCode...

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
